deployment/terraform/job-definitions/default.json

```diff
@@ -0,0 +1,16 @@
+{
+  "image": "${image_url}",
+  "vcpus": ${default_vcpus},
+  "memory": ${default_memory},
```

Comment:
Memory and vCPU limits need to be integers in job definitions, so the interpolation requires that these variables remain unquoted.